packages/carbon-graphs/src/js/controls/Graph/helpers/helpers.js

```diff
@@ -479,12 +482,22 @@ const updateAxesDomain = (config, input = {}) => {
 
   if (utils.notEmpty(input)) {
     const yAxis = input.config.yAxis || constants.Y_AXIS;
+    const xAxis = input.config.xAxis || constants.X_AXIS;
     config.axis[yAxis].domain = setDomain(
       config.outlierStretchFactor,
       config.axis[yAxis].domain.lowerLimit,
       config.axis[yAxis].domain.upperLimit,
       yAxis,
     );
+
+    if (config.axis.x.allowCallibration) {
+      config.axis[xAxis].domain = setDomain(
+        config.outlierStretchFactorXAxis,
+        config.axis[xAxis].lowerLimit,
+        config.axis[xAxis].upperLimit,
```

Comment:
Should these be `config.axis[xAxis].domain.lowerLimit` and `config.axis[xAxis].domain.upperLimit`?

Reply:
Fixed here: https://github.com/cerner/terra-graphs/pull/285/commits/e22ef8fc552d7a8674741d260737a0d8c9bc984b